Bump live-sync to v0.4.0 and document new flags (#4621)
timescale/docs: _partials/_livesync-terminal.md

   `--table-map`: (Optional) A JSON string that maps source tables to target tables. If not provided, the source and target table names are assumed to be the same.
   For example, to map the source table `metrics` to the target table `metrics_data`:

   `--table-sync-workers`: (Optional) The number of parallel workers to use for initial table sync. Default is 4.

>    `--copy-data`: (Optional) If specified, the initial table data is copied from source to target before starting logical replication. If not specified, only changes after the replication slot creation are replicated. Default is true.

Reviewer: ```suggestion
   `--copy-data`: (Optional) By default, the initial table data is copied from source to target before starting logical replication. Set to `false` so only changes made after replication slot creation are replicated.
```

>    Useful to dry-run the tool without copying data by setting it to false.

Reviewer: ```suggestion
   Best practice is to set to `false` during dry-run livesync so you do not copy table data.
```
